module/bdev/nvme/bdev_nvme.c

```diff
@@ -3797,14 +3798,25 @@ bdev_nvme_create_poll_group_cb(void *io_device, void *ctx_buf)
 	}
 
 	period = spdk_interrupt_mode_is_enabled() ? 0 : g_opts.nvme_ioq_poll_period_us;
+	if (spdk_interrupt_mode_is_enabled() && (g_nvme_trtype == SPDK_NVME_TRANSPORT_TCP)) {
+		/* For TCP transport in interrupt mode, the IO queue must be polled periodically
+		 * to flush data. Since TCP transport does not automatically push data to
+		 * the OS stack, we poll periodically to ensure timely processing of IO
+		 * commands.
+		 */
+		period = 100;
```

Comment:
What is the unit?

Reply:
Unit is in milliseconds.
https://github.com/spdk/spdk/blob/3cb3145bfa22a650a01c1183332a9f8c5bd2f868/doc/jsonrpc.md?plain=1#L3995